index.js

```diff
@@ -53,22 +55,21 @@ async function deployToEcs(deploy_file, version) {
     process.env.NEW_RELIC_LICENSE_KEY = readFileSecret('./SECRET_NEW_RELIC_LICENSE_KEY.txt');
     process.env.NODE_AUTH_TOKEN = readFileSecret('./SECRET_NPM_TOKEN.txt');
 
-    await exec.exec('npm i @springworks/ecs-deployer@2.26.0');
```

Comment:
This is the change. Instead of installing the package inside the container, we simply use the one from node_modules.